integration/tests/basic/rpc/test_neon_rpc_base_calls.py

```diff
@@ -126,3 +133,124 @@ def test_neon_get_native_token_list(self, pytestconfig, json_rpc_client):
             if item["tokenName"] == "NEON":
                 assert item["tokenMint"] == pytestconfig.environment.spl_neon_mint
                 assert item["tokenChainId"] == hex(pytestconfig.environment.network_ids["neon"])
+
+    @pytest.mark.parametrize("tx_type", TransactionType)
+    def test_neon_estimate_gas_iterative_tx(
+        self,
+        json_rpc_client: JsonRPCSession,
+        accounts: EthAccounts,
+        web3_client: NeonChainWeb3Client,
```

Comment:
you have 
accounts and web3_client as class objects
you can use it as  self.accounts /self.web3_client